client/my-sites/plans-features-main/jetpack-faq-i5/index.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FoldableFAQ from 'calypso/components/foldable-faq';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const JetpackFAQi5: React.FC = () => {
+	const translate = useTranslate();
+	// We want to allow Jetpack Free users to contact support, even when it isn't not available
+	// for them, because it could push them into purchasing a product. To make this possible, we
+	// add two query parameters that let's show the support form to Jetpack Free users:
+	// 1. rel=support is what shows the contact form
+	// 2. hpi=1 stands for has purchase intent
+	const getHelpLink = ( context: string ) => {
```

Comment:
This function could benefit from being moved out of the component, so that there's only one instance of it.